sentences/fr/homeassistant_HassTimerStatus.yaml

```diff
@@ -62,6 +62,8 @@ intents:
           - "Il reste [encore] combien [de temps] [<appele>] {timer_name:name}"
           # Combien de temps pour la pizza
           - "[Il reste] [encore] combien de temps [<appele>] {timer_name:name}"
+          # Combien de temps avant de fermer la fenêtre
+          - "[Il reste] [encore] combien de temps avant de {timer_name:name}"
```

Comment:
I think this `de` needs more thought.
It works well for verbs `Combien de temps avant de fermer la fenêtre`

But for exmple
`Combien de temps avant la pizza`

At least I think we need a `[de|<le>]`

Reply:
`Avant d'enteindre...` 

And `d'` also